src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -140,6 +143,23 @@ def reset_arg(arg, value):
                 reset_arg(arg, '0')
             reset_arg('compression_algorithm', 'unset')
 
+    def _wait_for_pgs(self, pool):
+        current_pool = self._get(pool)
+        initial_pgs = int(current_pool['pg_placement_num']) + int(current_pool['pg_num'])
+        self._pg_wait_loop(current_pool, initial_pgs)
+
+    def _pg_wait_loop(self, pool, initial_pgs):
+        if 'pg_num_target' in pool:
```

Comment:
I'm missing any documentation here. What types are the method parameters and what are they for? This info would help reviewing and understanding the code.

Reply:
`pool` is the pool object and `initial_pgs` is a number. I will add the missing documentation - Thanks for the hint :)